edge-modules/TwinTester/DesiredPropertiesValidator.cs

```diff
@@ -68,7 +38,7 @@ public override async Task ValidateAsync()
             }
             catch (Exception e)
             {
-                this.Logger.LogInformation($"Failed call to module client get twin: {e}");
+                Logger.LogInformation($"Failed call to module client get twin: {e}");
```

Comment:
```suggestion
                Logger.LogInformation(e, "Failed call to module client get twin");
```

Reply:
I changed to LogError

---
In reply to: [357802059](https://github.com/Azure/iotedge/pull/2108#discussion_r357802059) [](ancestors = 357802059)